addressbook.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto2";
```

Comment:
By convention I think we should put all .proto files under src/main/proto so when you run "mvn package" it automatically finds all proto files and compiles them

Reply:
So maven will try find everything under src/main by default or do we set it so somewhere in the pom.xml?

Reply:
I think the protobuf plugin will by default look .proto files under src/main/proto folder